pkg/machine/qemu/options_linux.go

```diff
@@ -11,3 +11,8 @@ func getRuntimeDir() (string, error) {
 	}
 	return util.GetRuntimeDir()
 }
+
+// makeSymlink is a no-op for Linux
+func (m *MachineFile) makeSymlink(*string) error {
```

Comment:
Why is this a noop for linux, we have the same problem there just that the limit is 108 vs 104 on darwin.